article.html

```diff
@@ -0,0 +1,437 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Space Ripples Reveal Big Bang's Smoking Gun</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='stylesheet' type='text/css' media='screen' href='css/main.css'>
+  <link rel='stylesheet' type='text/css' media='screen' href='css/fonts.css'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <header>
+    <nav class="navbar">
+      <div class="left_items">
+        <a href="#">
+          <i class="fa fa-bars"></i>
+        </a>
+        <a href="#">
+          <i class="fa fa-search"></i>
+        </a>
+      </div>
+      <div class="centered_items">
+        <a href="https://www.nytimes.com/"> <img src="./images/nyt_logo.png" alt="New York Times Logo"></a>
+      </div>
+      <div class="right_items">
+        <a class="nav_button" href="https://www.nytimes.com/subscription?campaignId=797YR">LOG IN</a>
+        <a class="nav_button" href="https://www.nytimes.com/subscription?campaignId=797YR">SUBSCRIBE NOW</a>
+      </div>
+    </nav>
+  </header>
+  <main id="content">
+    <article id="story" class="articleContent">
+      <header class="image_center">
+<<<<<<< HEAD
+        <h1 class="main-tittle" id="articleHeadline">Space Ripples Reveal Big Bang's Smoking Gun</h1>
+=======
+        <h1 class="main-tittle" id="articleHeadline ">Space Ripples Reveal Big Bang's Smoking Gun</h1>
+        <div class="social-share">
+          <i class="fa fa-twitter social-icon"></i>
+          <i class="fa fa-facebook social-icon"></i>
+          <i class="fa fa-envelope social-icon"></i>
+          <i class="fa fa-reply social-icon"></i>
+        </div>
+>>>>>>> 3aa163e373b183d65c345eb1253f07b08283bf23
+        <div class="headline">
+          <figure>
```

Comment:
## Failed merge 
- Fix this merge section, because when you merge between your and your partner changes you must have accepted the income changes